src/Razor/test/Microsoft.NET.Sdk.Razor.Test/Microsoft.NET.Sdk.Razor.Test.csproj

```diff
@@ -43,6 +43,16 @@
       <_Parameter2>$(MSBuildThisFileDirectory)..\testapps\TestPackageRestoreSource</_Parameter2>
     </AssemblyAttribute>
 
+    <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
+      <_Parameter1>ArtifactsLogDir</_Parameter1>
+      <_Parameter2>$([MSBuild]::NormalizeDirectory('$(ArtifactsDir)', 'log', '$(_BuildConfiguration)'))</_Parameter2>
```

Comment:
What's `$(_BuildConfiguration)`? Can this use `$(Configuration)`?

Reply:
It's the value defined in the yml file, I believe they are the same, but this way there's no chance for error.

Reply:
There is no `$(_BuildConfiguration)` variable in the YAML and it would never exist in local builds. Fortunately `ArtifactsLogDir` is never used.

Please delete this `AssemblyMetadataAttribute`.

Reply:
Its _BuildConfig. This was a mistake on my part. `_BuildConfig` is what is used in the yaml file to define the artifact location, so that's why I'm using it. See https://github.com/aspnet/AspNetCore-Tooling/blob/master/azure-pipelines.yml#L97

Reply:
ArtifactsLogDir is being used https://github.com/aspnet/AspNetCore-Tooling/pull/1146/files#diff-0d1aa219ea5eebd5e966605c7fdb83e7R176-R177